calculator.js

```diff
@@ -1,5 +1,26 @@
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
+// const testinput = {
+//   num1: 3,
+//   num2: 0,
+//   operation: '/',
+// }
+
+function calculator(input) {
+  switch(input.operation) {
+    case 'add':
+    case '+':
+      return input.num1 + input.num2;
+    case 'subtract':
+    case '-':
+      return input.num1 - input.num2;
+    case 'multiply':
+    case '*':
+      return input.num1 * input.num2;
+    case 'divide':
+    case '/':
+      return input.num1 / input.num2;
```

Comment:
This would be a good place for a divide by zero check

Reply:
Sorry, I'm realizing now that JS does it's own divide by zero check and responds with Infinity so a divide by zero check isn't really necessary.
This code is however missing the edgecase requirement for checking that the number inputs are actually numbers.